Quorum/apis/git_api/git_manager.py

```diff
@@ -24,26 +24,36 @@ def __init__(self, customer: str) -> None:
         """
         self.customer = customer
         
-        self.customer_path = config.MAIN_PATH / self.customer / "modules"
-        self.customer_path.mkdir(parents=True, exist_ok=True)
+        self.customer_modules_path = config.MAIN_PATH / self.customer / "modules"
+        self.customer_modules_path.mkdir(parents=True, exist_ok=True)
         
-        self.repos = self._load_repos()
+        self.customer_review_module_path = config.MAIN_PATH / self.customer / "review_module"
+        self.customer_review_module_path.mkdir(parents=True, exist_ok=True)
 
-    def _load_repos(self) -> dict:
+        self.repos, self.review_repo = self._load_repos_from_file()
+
+    def _load_repos_from_file(self) -> tuple[dict[str, str], dict[str, str]]:
```

Comment:
We can move "with open(config.REPOS_PATH) as f: repos_data = json.load(f)" to init function and split this to 2 different functions.

Reply:
Mmm I'm not sure I understand the benefits or maybe I didn't understand what you meant

Reply:
I tend to agree with your approach but I think it's an overkill in this specific situation. It will make just make the code longer with no real readability benefits.